libpromises/evalfunction.c

```diff
@@ -2775,6 +2847,22 @@ static FnCallResult FnCallMapData(EvalContext *ctx, ARG_UNUSED const Policy *pol
         return FnFailure();
     }
 
+    if (mapdatamode && jqmode)
+    {
+        // TODO: make this customizable
```

Comment:
Well, some people may not have it in `/usr/bin` :)

Maybe it can be in `def.jq` so you can override it from `def.json`.